src/chainparams.cpp

```diff
@@ -259,6 +294,10 @@ class CRegTestParams : public CTestNetParams {
             0,
             0
         };
+
+        // Founders reward script expects a vector of 2-of-3 multisig addresses
+        vFoundersRewardAddress = { "2N2e2FRfP9D1dRN1oRWkH7pbFM69eGNAuQ4" };
+        assert(vFoundersRewardAddress.size() < consensus.nSubsidyHalvingInterval + consensus.SubsidySlowStartShift());
```

Comment:
This should be rewritten in terms of `GetLastFoundersRewardBlockHeight()`

Non-blocking.